tinker/templates/admin/redirects/home.html

```diff
@@ -164,12 +164,35 @@
                     input.val("/" + value); // add in a slash
                     value = input.val();
                 }
+                if (value[0] == '/' && value[1] =='/'){
+                    input.val(input.val().replace('//','/')); // eliminates a lead double slash
+                }
                 delay(function () {
                     loadTableForSearch(value, $('#new-redirect-to').val());
                     $('#spinner').hide();
                 }, 300);
             });
 
+            $('#modal-from-path-edit').keyup(function(){
+                var validRedirect = /(\/)[^\/\s][^\s]*/;
```

Comment:
this definitely needs a comment. Also, this isn't used?

Reply:
Oh yeah this was my first attempt at doing regular expression checking like Phil suggested. It didn't work, so I'll just delete that line.